src/services/management/accountMerchantLevelApi.ts

```diff
@@ -56,7 +56,7 @@ export default class AccountMerchantLevelApi extends Service {
      * @summary Create a merchant account
      * @param createMerchantRequest 
      */
-    public async postMerchants(createMerchantRequest: CreateMerchantRequest, requestOptions?: IRequest.Options): Promise<CreateMerchantResponse> {
+    public async createMerchantAccount(createMerchantRequest: CreateMerchantRequest, requestOptions?: IRequest.Options): Promise<CreateMerchantResponse> {
         const localVarPath = "/merchants";
         const resource = new ManagementResource(this, localVarPath);
         const request: CreateMerchantRequest = ObjectSerializer.serialize(createMerchantRequest, "CreateMerchantRequest");
```

Comment:
Do we need these implicit lines here serializing the request? Don't we already serialize in the actual getJsonResponse method?

Reply:
Correct, the JSON string is produced in `getJsonResponse`. This method `ObjectSerializer.serialize`, does other transformations like converting a `Date` to `string` in the format expected by the API.